GameData/KerbalismConfig/Support/EngineIgnitor.cfg

```diff
@@ -1,5 +1,5 @@
 // disable all engine failures
-@PART[*]:HAS[@MODULE[ModuleEngines*]]:NEEDS[EngineIgnitor]:AFTER[Kerbalism]
+@PART[*]:HAS[@MODULE[ModuleEngines*]]:NEEDS[EngineIgnitor]:NEEDS[!KiwiTechTree]:AFTER[Kerbalism]
```

Comment:
You can't have multiple NEEDS, this is invalid MM syntax.
You need to do something like : 
`@PART[*]:HAS[@MODULE[ModuleEngines*]]:NEEDS[EngineIgnitor,!KiwiTechTree]:AFTER[Kerbalism]`